calcom/cal.com
# fix: fixes event types toLowerCase issue on save

## What does this PR do?



Fixes #9790



https://github.com/calcom/cal.com/assets/47187878/714b0f83-0164-4483-939d-05ce53c44699

- I have updated the slugify function to accommodate for the slugs where we don't need to convert the slug to lowercase.
- Have done these updates to all the places where event type slug is being created or updated.

## Type of change



- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?



- Create an event type with a slug containing capital letters
- The event type slug should not turn the slug toLowerCase in both the cases while creating and editing

## Mandatory Tasks
